src/Polly.Extensions/Telemetry/TelemetryListenerImpl.cs

```diff
@@ -43,7 +43,13 @@ public TelemetryListenerImpl(TelemetryOptions options)
 
     public override void Write<TResult, TArgs>(in TelemetryEventArguments<TResult, TArgs> args)
     {
-        _listener?.Write(in args);
+        if (_listeners.Count > 0)
+        {
+            foreach (var listener in _listeners)
+            {
+                listener.Write(in args);
+            }
```

Comment:
As this is a `List<T>`, would using its own `ForEach()` method be more performant than using a `foreach`?

```suggestion
            _listeners.ForEach((listener) => listener.Write(in args));
```

Reply:
It would closure allocation because of `args`.